webapp/src/components/viewHeader/viewHeaderPropertiesMenu.test.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {ReactElement} from 'react'
+import {render, screen} from '@testing-library/react'
+import {Provider as ReduxProvider} from 'react-redux'
+import configureStore from 'redux-mock-store'
+
+import '@testing-library/jest-dom'
+import userEvent from '@testing-library/user-event'
+
+import {IntlProvider} from 'react-intl'
+
+import {TestBlockFactory} from '../../test/testBlockFactory'
+
+import ViewHeaderPropertiesMenu from './viewHeaderPropertiesMenu'
+
+const wrapIntl = (children: ReactElement) => (
+    <IntlProvider locale='en'>{children}</IntlProvider>
+)
+const board = TestBlockFactory.createBoard()
+const activeView = TestBlockFactory.createBoardView(board)
+
+describe('components/viewHeader/viewHeaderPropertiesMenu', () => {
+    const state = {
+        users: {
+            me: {
+                id: 'user-id-1',
+                username: 'username_1'},
+        },
+    }
+    const mockStore = configureStore([])
+    const store = mockStore(state)
```

Comment:
Looks like this mock store will be required in many unit tests. So it will be better to create a function for this piece of code and add it to [testUtils.tsx](https://github.com/mattermost/focalboard/blob/main/webapp/src/testUtils.tsx) which has already similar functions.

Reply:
I did a function in testUtils for that 😉